README.md

```diff
@@ -10,6 +10,9 @@ functions on [AWS Lambda](https://aws.amazon.com/lambda/). Bender Core handles
 the complex plumbing and provides the interfaces necessary to build modules for
 all aspects of the ETL process.
 
+### Configuration
+
+Bender is easily configurable with either json or yaml. The [configuration guide](http://oss.nextdoor.com) provides documentation for option specifics and [sample_configs](https://github.com/Nextdoor/bender/blob/master/sample_configs) contains real world examples of how Bender is configured with commonly used pipelines.
```

Comment:
Line-wrap @ 80 chars pls?
Whats the link to http://oss.nextdoor.com? That site hasn't worked in years?